src/NetAnalyzers/CSharp/Microsoft.NetCore.Analyzers/InteropServices/CSharpPreferIsExactSpellingIsTrueForKnownApis.Fixer.cs

```diff
@@ -0,0 +1,164 @@
+// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Composition;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Analyzer.Utilities;
+using Analyzer.Utilities.Extensions;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CodeFixes;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Microsoft.CodeAnalysis.Editing;
+using Microsoft.NetCore.Analyzers;
+using Microsoft.NetCore.Analyzers.InteropServices;
+
+namespace Microsoft.NetCore.CSharp.Analyzers.InteropServices
+{
+    /// <summary>
+    /// CA1839: Prefer ExactSpelling=true for known Apis fixer
+    /// </summary>
+    [ExportCodeFixProvider(LanguageNames.CSharp, Name = PreferIsExactSpellingIsTrueForKnownApisAnalyzer.RuleId), Shared]
+    public sealed class CSharpPreferIsExactSpellingIsTrueForKnownApisFixer : CodeFixProvider
+    {
+        private const string ExactSpellingText = "ExactSpelling";
+        private const string EntryPointText = "EntryPoint";
+        internal const string RuleId = "CA1839";
+
+        public override ImmutableArray<string> FixableDiagnosticIds => ImmutableArray.Create(PreferIsExactSpellingIsTrueForKnownApisAnalyzer.RuleId);
+
+        private static SyntaxNode? FindNamedArgument(IReadOnlyList<SyntaxNode> arguments, string argumentName)
```

Comment:
Could be `IEnumerable<SyntaxNode>`

Reply:
I do not think arguments can have the same name?

Reply:
Not sure to get what you are asking.

Reply:
Ah you are referring to the parameter type?